test/unit/lodash.spec.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function ()
+    {
+        describe('clamp', function ()
+        {
+
+            var params, number, lower, upper;
```

Comment:
As long as we will have only one **it** per **describe** we don't even need to have this variables, because we can test params strictly getting it by [] notation.